hivemq-edge/src/main/java/com/hivemq/configuration/entity/adapter/NorthboundMappingEntity.java

```diff
@@ -242,28 +266,14 @@ public String toString() {
                 '}';
     }
 
-    public static @NotNull NorthboundMappingEntity fromPollingContext(PollingContext ctx) {
-        final List<MqttUserPropertyEntity> mqttUserProperties = ctx.getUserProperties()
-                .stream()
-                .map(mqttUserPropertyEntity -> new MqttUserPropertyEntity(mqttUserPropertyEntity.getName(),
-                        mqttUserPropertyEntity.getValue()))
-                .collect(Collectors.toList());
-
-        return new NorthboundMappingEntity(
-                ctx.getTagName(),
-                ctx.getMqttTopic(),
-                ctx.getMqttQos(),
-                ctx.getMessageHandlingOptions(),
-                ctx.getIncludeTagNames(),
-                ctx.getIncludeTimestamp(),
-                mqttUserProperties,
-                ctx.getMessageExpiryInterval());
-    }
-
     @Override
     public boolean equals(final Object o) {
```

Comment:
@Nullable Object o

Reply:
There are too many of them all over the code base. I'm out of bandwidth this time.